### PR TITLE
Add html comp

### PR DESCRIPTION
**Pull request by bhohnco**
## Is this a fix or a feature?
Finished up the basic HTML for the comp/iteration 0.
## What is the change?
Added button functions and text input boxes.
## What does it fix?
It adds more options to the main site. 
## Where should the reviewer start?
index.html line 30
## How should this be tested?
Opening the webpage on your local machine to see the changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
-    <main>
-  <div class ="header1"><h2>New Activity</h2>
-  </div>
-  <section class="boxes" id="box1">
-      <h2 class="box1-lead">Select a category:</h2>
-      <br>
-      <div class="category">
-        <button class="category" id="study" type="button">Study</button>
-        <button class="category" id="meditate" type="button">Meditate</button>
-        <button class="category" id="exercise" type="button">Exercise</button>
```
Comment (percworld) [inline index.html:19]: End section?

File: index.html [hunk 2]
```diff
@@ -1,29 +1,49 @@
...
+<header class ="header">
+    <h1>INTENTION TIMER</h1>
+</header>
+<main>
```
Comment (daomeow) [inline index.html:12]: The following main tag doesn't quite line up

File: index.html [hunk 3]
```diff
@@ -1,29 +1,49 @@
...
+        <h2 class="box1-lead">Select a category:</h2>
+        <br>
+        <div class="category">
+            <button class="category" id="study" type="button">Study</button>
+            <button class="category" id="meditate" type="button">Meditate</button>
+            <button class="category" id="exercise" type="button">Exercise</button>
+        </div>
+        <form>
+            <div>
+                <label class="description" id="accomplish">
```
Comment (percworld) [inline index.html:26]: Change ID to description

File: index.html [hunk 4]
```diff
@@ -1,29 +1,49 @@
...
+                    <h1>What would you like to accomplish during this time?</h1>
+                    <input class="accomplish-time" id="accomplish-t" type="text"/>
```
Comment (percworld) [inline index.html:28]: Semantics attention -t

**Review by percworld — APPROVED** [review]
ID change and Semantics

**Review by daomeow — APPROVED** [review]

**Pull request merged**